python/common/org/python/types/Bytes.java

```diff
@@ -170,10 +170,31 @@ public Bytes(org.python.Object[] args, java.util.Map<java.lang.String, org.pytho
     }
 
     @org.python.Method(
-            __doc__ = "Return key in self."
-    )
-    public org.python.Object __contains__(java.util.List<org.python.Object> args, java.util.Map<java.lang.String, org.python.Object> kwargs, java.util.List<org.python.Object> default_args, java.util.Map<java.lang.String, org.python.Object> default_kwargs) {
-        throw new org.python.exceptions.NotImplementedError("bytes.__contains__ has not been implemented.");
+            __doc__ = "Return true if contains the arg",
+            args = {"slice"}
+    )
+    public org.python.Object __contains__(org.python.Object slice) {
+        if (slice instanceof org.python.types.Bytes) {
+            int counter_slice = 0;
+            for (int i = 0; i < this.value.length +1; i++){
+                if (counter_slice == ((org.python.types.Bytes) slice).value.length){
+                    return new org.python.types.Bool(true);
+                }
+                else if (i == this.value.length){
+                    break;
+                }
+                else if (((org.python.types.Bytes) slice).value[counter_slice] == this.value[i]){
+                    counter_slice++;
+                }
+                else {
+                    counter_slice = 0;
+                }
+            }
+            return new org.python.types.Bool(false);
+        }
+        else {
+              throw new org.python.exceptions.TypeError("invalid type for this method");
```

Comment:
The exception messages should match the messages that CPython does.
CPython (testing on 3.5.2) throws a different message, see:

```
>>> ['p', 'y'] in b'pybee'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: a bytes-like object is required, not 'list'
```

You can build the message like it's being done here: https://github.com/pybee/voc/blob/9e0dca72dc874249977a11ea40bf112f7af2485f/python/common/org/python/types/Bytes.java#L667